storage/rocksdb/rdb_datadic.h

```diff
@@ -387,7 +387,11 @@ class Rdb_key_def
     SECONDARY_FORMAT_VERSION_INITIAL= 10,
     // This change the SK format to include unpack_info.
     SECONDARY_FORMAT_VERSION_UPDATE1= 11,
-    SECONDARY_FORMAT_VERSION_LATEST= SECONDARY_FORMAT_VERSION_UPDATE1,
```

Comment:
Since indices with prefix columns are already broken and cannot be used correctly by queries, it might be fine to fix the code without updating the key version. This would simplify the diff and testcase. Backward compatibility shouldn't be needed since there are probably no existing installations relying on it because it never worked correctly in the first place.